src/Avalonia.FreeDesktop/Avalonia.FreeDesktop.csproj

```diff
@@ -12,7 +12,8 @@
   <ItemGroup>
```

Comment:
You now can import TrimmingEnable props file, so we will see if there are any warnings in the project.
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Avalonia.Base.csproj#L21